src/test/java/com/github/fge/jsonpatch/query/AddQueryOperationTest.java

```diff
@@ -0,0 +1,12 @@
+package com.github.fge.jsonpatch.query;
+
+import com.github.fge.jsonpatch.JsonPatchOperationTest;
+
+import java.io.IOException;
+
+public class AddQueryOperationTest extends Object {
+
+    public AddQueryOperationTest() throws IOException {
+        //super("query/add");
```

Comment:
do we need it? (similar in other `*OperationTest classes)

Reply:
Commented above